examples/accordion/accordion.html

```diff
@@ -245,33 +265,15 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
         </tr>
       </thead>
       <tbody>
-        <tr>
-          <th scope="row"><code>presentation</code></th>
-          <td></td>
-          <td><code>dl</code></td>
-          <td>
-            <ul>
-              <li>Indicates that the <code>dl</code> element is being used to control presentation and does not represent a definition list.</li>
-              <li>This implementation uses an HTML definition list where each term is recast as a header and each definition is recast as a region that contains panel content.</li>
-            </ul>
-          </td>
-        </tr>
         <tr>
           <th scope="row"><code>heading</code></th>
           <td></td>
-          <td><code>dt</code></td>
-          <td>Identifies the element as a heading that serves as an accordion header.</td>
-        </tr>
-        <tr>
-          <td></td>
-          <th scope="row"><code>aria-level=<q>3</q></code></th>
-          <td><code>dt</code></td>
+          <td><code>h3</code></td>
           <td>
             <ul>
-              <li>Specifies heading level for the accordion header.</li>
-              <li>Level 3 is chosen because the accordion is contained in a section with a level 2 heading.</li>
-            </ul>
-          </td>
+              <li>The accordion design pattern is defined by a button is contained within an element with <code>role=heading</code>.</li>
+              <li>The default role for the <code>h3</code> element is <code>heading</code>.</li>
```

Comment:
This sentence: "The accordion design pattern is defined by a button is contained within an element with role=heading."
- either needs "that" added after "button", i.e. "...a button that is contained within..."
- OR the "is" after "button" can be deleted, i.e. "...a button contained within..."